### PR TITLE
FEAT: basic `form/into` and `mold/into` implementation (appends only)

### PR DESCRIPTION
**Pull request by Oldes**

### CODE REVIEW

File: runtime/actions.reds [hunk 1]
```diff
@@ -251,7 +252,11 @@ actions: context [
 		][0]
 		
 		stack/keep										;-- keep last value
-		buffer: string/rs-make-at stack/push* 16		;@@ /part argument
+		either into >= 0 [
+			buffer: as red-string! stack/arguments + into
+		][
+			buffer: string/rs-make-at stack/push* either limit = 0 [16][limit]
```
Comment (dockimbel) [inline runtime/actions.reds]: You should not nest `either` in expressions, the R/S compiler has very little support for that for now.

File: runtime/actions.reds [hunk 2]
```diff
@@ -301,7 +307,11 @@ actions: context [
 		][0]
 
 		stack/keep										;-- keep last value
-		buffer: string/rs-make-at stack/push* 16		;@@ /part argument
+		either into >= 0 [
+			buffer: as red-string! stack/arguments + into
+		][
+			buffer: string/rs-make-at stack/push* either limit = 0 [16][limit]
```
Comment (dockimbel) [inline runtime/actions.reds]: Same remark as the previous one.

**Comment by Oldes** [comment]
Unfortunately there is a problem when used with function:
```
f: does [] s: copy "" mold/into :f s

*** Runtime Error 101: no value matched in SWITCH
*** in file: /x/GIT/Red/runtime/datatypes/string.reds
*** at line: 465
```

**Comment by greggirwin** [comment]
@oldes, your last comments indicate a problem with the code. Is that something you want to fix, or should we close this PR and revisit, maybe along with Doc's notes on `reduce/into`?

**Comment by greggirwin** [comment]
@Oldes, I'm going to close this PR if there isn't any planned progress on it.

**Comment by greggirwin** [comment]
Closing, due to merge conflicts and inactivity.

**Pull request closed without merging**